op-deployer/pkg/deployer/opcm/proxy.go

```diff
@@ -0,0 +1,67 @@
+package opcm
+
+import (
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/common"
+
+	"github.com/ethereum-optimism/optimism/op-chain-ops/script"
+)
+
+type DeployProxyInput struct {
+	Owner common.Address
+}
+
+func (input *DeployProxyInput) InputSet() bool {
+	return true
+}
+
+type DeployProxyOutput struct {
+	Proxy common.Address
+}
+
+func (output *DeployProxyOutput) CheckOutput(input common.Address) error {
```

Comment:
Rather than doing this here in the I/O contract, do it in the script contract itself.